app/javascript/packs/containers/NavigationBar/NavigationBarContainer.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { getBuildings, getBuildingById } from '../../selectors/buildingsSelector';
+import { connect } from 'react-redux';
+import DropdownMenuContainer from './DropdownMenuContainer'
+import { getName, getEmail } from '../../selectors/usersSelector'
+import {
+  getCurrentCategory,
+  getCategoriesForBuilding,
+} from "../../selectors/categoriesSelector";
+import CategoryContainer from './CategoryContainer';
+import { getQuestionsByBuilding } from "../../selectors/questionsSelector";
+import { getRemainingAnswersforCategory } from "../../selectors/answersSelector";
+import { getQuestionsByCategory } from "../../utils/QuestionsFilter";
+
+
+class NavigationBarContainer extends React.Component {
+
+  render() {
+    const username = this.props.username;
+    const userEmail = this.props.userEmail;
+    return (
+      <div>
+        <DropdownMenuContainer
```

Comment:
Nit: since `DropdownMenuContainer` and `CategoryContainer` don't have children, you can use self-closing tags, i.e. `<DropdownMenuContainer {...} />`